gluten-ut/src/test/scala/org/apache/spark/sql/GlutenTestsBaseTrait.scala

```diff
@@ -27,40 +28,29 @@ trait GlutenTestsBaseTrait {
   protected val warehouse: String = basePath + "/spark-warehouse"
   protected val metaStorePathAbsolute: String = basePath + "/meta"
 
-  def whiteTestNameList: Seq[String] = Seq.empty
+  // prefer to use testNameBlackList
+  def testNameBlackList: Seq[String] = Seq()
 
-  // prefer to use blackTestNameList
-  def blackTestNameList: Seq[String] =
-    NotSupport.NotYetSupportCase(getClass.getSuperclass.getSimpleName)
-
-  def whiteBlackCheck(testName: String): Boolean = {
-    if (testName.startsWith(GlutenTestConstants.GLUTEN_TEST)) {
-      true
-    } else if (blackTestNameList.isEmpty && whiteTestNameList.isEmpty) {
-      true
-    } else if (blackTestNameList.nonEmpty &&
-               blackTestNameList.head.equalsIgnoreCase(GlutenTestConstants.IGNORE_ALL)) {
-      false
-    } else if (blackTestNameList.nonEmpty) {
-      val exactContain = blackTestNameList.contains(testName)
-      if (exactContain) return false
-
-      // some test cases' names start with SPARK-ISSUE_ID:
-      // we allow simply put SPARK-ISSUE_ID in blacklist for short
-      var fuzzyContain = false
-      if (testName.startsWith("SPARK-")) {
-        import scala.util.matching.Regex
-        val issueIDPattern = "SPARK-[0-9]+".r
-        val issueID = issueIDPattern.findFirstIn(testName) match {
-          case Some(x: String) => x
-        }
-        fuzzyContain = blackTestNameList.exists(_.startsWith(issueID))
+  def shouldRun(testName: String): Boolean = {
+    if (testName.startsWith(GLUTEN_TEST)) {
+      return true
+    }
+    if (testNameBlackList.exists(_.equalsIgnoreCase(GlutenTestConstants.IGNORE_ALL))) {
```

Comment:
Is exclude case should add to testNameBlackList

Reply:
No need to do that. And I added following comment to `testNameblackList`

https://github.com/oap-project/gluten/blob/558f35a6e4e82eda9d074384f512584a7b767d6c/gluten-ut/src/test/scala/org/apache/spark/sql/GlutenTestsBaseTrait.scala#L31-L33

Does the comment look clear enough to you?